### PR TITLE
[Merged by Bors] - chore(Topology/ContinuousFunction/Bounded): add a `BoundedSMul` instance

### PR DESCRIPTION
**Pull request by eric-wieser**
This typeclass probably didn't exist yet when this was written.

Also cleans up some nearby style and comments.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/ContinuousFunction/Bounded.lean [hunk 1]
```diff
@@ -1401,32 +1401,31 @@ functions from `α` to `β` is naturally a module over the algebra of bounded co
...
 
-theorem norm_smul_le (f : α →ᵇ 𝕜) (g : α →ᵇ β) : ‖f • g‖ ≤ ‖f‖ * ‖g‖ :=
-  norm_ofNormedAddCommGroup_le _ (mul_nonneg (norm_nonneg _) (norm_nonneg _)) _
-#align bounded_continuous_function.norm_smul_le BoundedContinuousFunction.norm_smul_le
-
-/- TODO: When `NormedModule` has been added to `Analysis.NormedSpace.Basic`, the above facts
-show that the space of bounded continuous functions from `α` to `β` is naturally a normed
+/- TODO: When `NormedModule` has been added to `Analysis.NormedSpace.Basic`, this
+shows that the space of bounded continuous functions from `α` to `β` is naturally a normed
 module over the algebra of bounded continuous functions from `α` to `𝕜`. -/
```
Comment (j-loreaux) [inline Mathlib/Topology/ContinuousFunction/Bounded.lean:1423]: Are we ever doing this? I thought we were purposefully avoiding it.

bors d+

Comment (eric-wieser) [reply Mathlib/Topology/ContinuousFunction/Bounded.lean:1423]: Yeah, I suspect we're not, but I wasn't really sure what to do with this comment and wanted to just declare fixing it out of scope.

Comment (eric-wieser) [reply Mathlib/Topology/ContinuousFunction/Bounded.lean:1423]: Do you want me to delete it?

**Comment by eric-wieser** [comment]
bors merge

I can delete the comment in a follow-up if you want me to

**Pull request closed without merging**